src/tests/Microsoft.PowerFx.Interpreter.Tests/PowerFxEvaluationTests.cs

```diff
@@ -385,7 +421,16 @@ protected override async Task<RunResult> RunAsyncInternal(string expr, string se
                                 config = k.initPfxConfig(config);
                             }
 
-                            object o = k.updatePfxConfig?.Invoke(config);
+                            object o = k.updatePfxConfig?.Invoke(config, symbolTable);
+
+                            if (o is ISymbolSlot slot)
+                            {
+                                slots.Add(slot);
+                            }

```

Comment:
I don't see this codepath being hit. If it isn't, let's not add it. And if you pass the result of updatePfxConfig to setVariables, you won't need to define those variables here.